### ISSUE TITLE
Android ContextMenu Change Colors

### ISSUE THREAD

**Issue by SalihColak** [mpiannucci/react-native-context-menu-view#108]
Is there any way to change the background and the text color of the ContextMenu on Android? In IOS it is automatically changed when switching to light/dark mode.

**Comment by MRadek55** [comment]
+1

**Comment by impe93** [comment]
Same problem here, I would like to have the possibility to change default colors on Android.

At the moment the situation is the following, and it's not good looking.

<img width="773" alt="Screenshot 2024-03-16 alle 12 37 47" src="https://github.com/mpiannucci/react-native-context-menu-view/assets/25618834/208f5df1-33b3-4fd0-bd8f-f877c0272d6e">

**Comment by mpiannucci** [comment]
on iOS the context menu follows the system theme automatically. Is there a way to do the same on Android?

**Comment by MrAdex77** [comment]
+1 this is really needed

**Comment by impe93** [comment]
> on iOS the context menu follows the system theme automatically. Is there a way to do the same on Android?

I'm going to check when I have some spare time mate, if I get something I'll let you know or I'm going to open a PR to fix 👍

**Comment by DavichoStar** [comment]
Any news with this? Or a temporary solution?

**Comment by tarnumius** [comment]
> Any news with this? Or a temporary solution?

You can disable dark theme in you app by change styles.xml

Change line     <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar"> to     <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">

**Comment by vafada** [comment]
seems to work for me w/o changing anything.

Using the example app

light theme:

![Screenshot from 2024-09-21 18-49-35](https://github.com/user-attachments/assets/ee41b25e-a37d-4440-8e2d-5de246eb02aa)

dark theme:
![Screenshot from 2024-09-21 18-53-23](https://github.com/user-attachments/assets/b5770dd1-7aab-4ff4-9684-0267b9f34125)

**Comment by atambo** [comment]
It would be nice if the ContextMenu would follow the react native Appearance color scheme override instead of always using the system color scheme: https://reactnative.dev/docs/appearance#setcolorscheme

That way apps can implement a light/dark theme toggle and have the ContextMenu adapt to the above setting.

**Comment by baudc** [comment]
I agree with the neighbor above, many apps include a setting to follow system mode or force dark mode. Imho, it is very much needed.
